src/goModules.ts

```diff
@@ -59,6 +59,17 @@ export function getModFolderPath(fileuri: vscode.Uri): Promise<string> {
 					goConfig.update('formatTool', 'goimports', vscode.ConfigurationTarget.WorkspaceFolder);
 					vscode.window.showInformationMessage('`goreturns` doesnt support auto-importing missing imports when using Go modules yet. So updating the "formatTool" setting to `goimports` for this workspace.');
 				}
+				if (goConfig['useLanguageServer'] === false) {
```

Comment:
so by defualt this is false or true? if its true and someone is manually setting it to false, do we want to prompt users with this ?

Reply:
The default is false.